win32/Lib/winerror.py

```diff
@@ -1,64 +1,219 @@
 """Error related constants for win32
 
-Generated by h2py from winerror.h
-"""
+Generated from winerror.h and cderror.h"""
```

Comment:
why is cderror now included?

Reply:
It already was, see this block from main: https://github.com/mhammond/pywin32/blob/71c601767bd029fe918c81451c4504ecef138b39/win32/Lib/winerror.py#L3003-L3040